understanding/focus-visible.html

```diff
@@ -35,7 +35,7 @@ <h1>達成基準 2.4.7: フォーカスの可視化を理解する</h1>
             <h2>意図</h2>
             <p>この達成基準の意図は、キーボード フォーカスを持っている要素を利用者が認識しやすくすることである。</p>
             <p>この達成基準の意図は、複数の要素のうち、どの要素がキーボード フォーカスを持っているか利用者が認識しやすくすることである。スクリーン上にキーボード操作可能な要素が一つだけある場合には、視覚的にはキーボード操作可能な要素を一つだけ提示するため、達成基準が満たされることになる。</p>
-            <p>注記: キーボードのフォーカスインジケータは異なる形態をとることができる点に注意すべきである。一般的な方法の一つとして、テキストフィールドがキーボード フォーカスを持っていることを示すための、テキストフィールド内のキャレット表示がある。別の方法としては、ボタンがキーボード フォーカスを持っていることを示すために、ボタンを視覚的に変化させるといった方法がある。</p>
+            <p>注記: キーボードのフォーカスインジケータは異なる形態をとることができる点に注意すべきである。</p>
```

Comment:
```suggestion
            <p>キーボードのフォーカスインジケータは異なる形態をとることができる点に注意する。</p>
```

> Note that a keyboard focus indicator can take different forms.

なので、「注記:」というわけではないかなと。

Reply:
ホントですね。「注記:」は消します！